### PR TITLE
adding recordings

### PR DESCRIPTION
**Pull request by l0lawrence**
In this PR:

- Renaming Samples SERVICE_BUS to be SERVICEBUS to match the ARM resource template
- Creating new recordings for all mgmt_tests using test proxy and deleting the old recordings
- Moving mgmt_test location in the tests/ folder
- Updating the conftest.py to use test proxy   
#26508

### CODE REVIEW

File: sdk/servicebus/azure-servicebus/samples/async_samples/authenticate_client_connstr_async.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 RES_GROUP=<resource-group-name>
 NAMESPACE_NAME=<servicebus-namespace-name>
 
-export SERVICE_BUS_CONN_STR=$(az servicebus namespace authorization-rule keys list --resource-group $RES_GROUP --namespace-name $NAMESPACE_NAME --name RootManageSharedAccessKey --query primaryConnectionString --output tsv)
+export SERVICEBUS_CONN_STR=$(az servicebus namespace authorization-rule keys list --resource-group $RES_GROUP --namespace-name $NAMESPACE_NAME --name RootManageSharedAccessKey --query primaryConnectionString --output tsv)
```
Comment (catalinaperalta) [inline sdk/servicebus/azure-servicebus/samples/async_samples/authenticate_client_connstr_async.py:18]: Where is this used?

**Review by kashifkhan — APPROVED** [review]
🚀

File: sdk/servicebus/azure-servicebus/conftest.py [hunk 2]
```diff
@@ -3,15 +3,35 @@
...
+)
+
 
 collect_ignore = []
 
+@pytest.fixture(scope="session", autouse=True)
+def add_sanitizers(test_proxy):
+    # set_custom_default_matcher(
+    #     compare_bodies=False,
+    # )
```
Comment (mccoyp) [inline sdk/servicebus/azure-servicebus/conftest.py:24]: nit: we can remove this bit 🙂

File: sdk/servicebus/tests.yml [hunk 3]
```diff
@@ -5,15 +5,15 @@ stages:
     parameters:
       ServiceDirectory: servicebus
       TestTimeoutInMinutes: 300
-      MatrixReplace:
-        - TestSamples=.*/true
+      TestProxy: true
```
Comment (mccoyp) [inline sdk/servicebus/tests.yml:8]: The test proxy isn't used in live test pipelines, so we can actually remove this parameter

**Comment by mccoyp** [comment]
This is awesome, thank you! Does this PR replace/obviate https://github.com/Azure/azure-sdk-for-python/pull/23923?

**Review by mccoyp — APPROVED** [review]

**Pull request merged**